src/Compute/Compute/help/Set-AzVMADDomainExtension.md

```diff
@@ -21,7 +21,7 @@ Set-AzVMADDomainExtension -DomainName <String> [-OUPath <String>] [-JoinOption <
 ```
 
 ## DESCRIPTION
-The **Set-AzVMADDomainExtension** cmdlet adds an Azure Active Directory (AD) domain virtual machine extension to a virtual machine.
+The **Set-AzVMADDomainExtension** cmdlet adds a Microsoft Entra ID \(AD) domain virtual machine extension to a virtual machine.
```

Comment:
Same issue. The added backslash `\` is unnecessary and `(AD)` should be removed.